lux/core/frame.py

```diff
@@ -127,14 +122,25 @@ def expire_metadata(self):
         # Set metadata as null
         self._metadata_fresh = False
         self.data_type_lookup = None
-        self.data_type = None
-        self.data_model_lookup = None
-        self.data_model = None
         self.unique_values = None
         self.cardinality = None
         self._min_max = None
         self.pre_aggregated = None
 
+    def compute_data_type_from_lookup(self):
```

Comment:
Why is this inside the `frame.py` instead of the `Executor.py`(which is still shared across `SQLExecutor` and `PandasExecutor`)? I think that earlier this semester @thyneb19 worked on moving data type related stuff outside of `frame.py` so maybe we should keep it that way?

Reply:
Great, I'll move it over